app/views/users/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<div>
+  <%= @user.name %>
+</div>
```

Comment:
- [ ] Please add a placeholder explaining what this view show instead of showing the user's name because it's a project requirement.

![image](https://user-images.githubusercontent.com/68796232/184678933-6c0e6549-46ca-4a69-90d2-94f08f6e4b05.png)

[Hint]: you will edit these views later in the upcoming milestones.